libmamba/src/core/util_string.cpp

```diff
@@ -554,5 +635,10 @@ namespace mamba
         {
             return 1;
         }
+        std::size_t size(const wchar_t c)
+        {
+            return 1;
```

Comment:
Hmm is this supposed to return the size of `wchar_t`? If so, it's wrong. Otherwise, I would say that we should use `sizeof(wchar_t)` instead of returning a hardcoded value.
(I also realized that there is a `return 1;` for `char` case in the same file in `master`, to be changed to `sizeof(char)`).

Reply:
It's supposed to return the size of a string. Since some functions work with both a string or a character, we use this generic overload to view a characters as a 1-character strings.

For instance in `concatenate` we can concatenate `string` or `char`. We compute the final size by calling this functions on all inputs.

Reply:
Hmm ok, yeah so maybe rename it `str_size` or `str_len` or something similar?